ci/install-package.sh

```diff
@@ -78,12 +88,18 @@ echo -e "\e[35m\e[1m Trying to fetch docker image: $IMAGE_NAME:$BRANCH_TAG \e[0m
 if ! docker pull "$IMAGE_NAME:$BRANCH_TAG"
 then
     echo -e "\e[35m\e[1m No worries, we just test against the master branch: $IMAGE_NAME:$MASTER_TAG \e[0m"
-    docker pull $IMAGE_NAME:$MASTER_TAG
-    BRANCH_TAG=master
+    docker pull "$IMAGE_NAME":"$MASTER_TAG"
+    BRANCH_TAG=$MASTER_TAG
 fi
 
 # Run the docker image along with setting new environment variables
-docker run --detach --interactive -e CI=true -e PACKAGE="$PACKAGE" -e BRANCH="$BRANCH" -e COMMIT="$COMMIT" -e PULL_REQUEST="$PULL_REQUEST" --name tue-env "$IMAGE_NAME:$BRANCH_TAG"
+docker run --detach --interactive -e CI="true" -e PACKAGE="$PACKAGE" -e BRANCH="$BRANCH" -e COMMIT="$COMMIT" -e PULL_REQUEST="$PULL_REQUEST" --name tue-env "$IMAGE_NAME:$BRANCH_TAG"
+
+if [ "$USE_SSH" != "false" ]
```

Comment:
```suggestion
if [ "$USE_SSH" == "true" ]
```
This is a better way as it forces people to do it right

Reply:
Updated this to be just a flag.